src/window.cpp

```diff
@@ -3085,25 +3123,25 @@ void UpdateWindows()
 	 * But still empty the invalidation queues above. */
 	if (_network_dedicated) return;
 
-	static int we4_timer = 0;
-	int t = we4_timer + 1;
+	static int hundredth_timer = 1;
+	if (TimerElapsed(hundredth_timer, delta_ms)) {
+		hundredth_timer = 3000; // Historical reason: 100 * MILLISECONDS_PER_TICK
```

Comment:
why not define it as 100 * MILLISECONDS_PER_TICK ?

Reply:
Because it's no longer really linked to M_P_T.